packages/manager/modules/manager-pci-common/src/components/rclone-download/S3StorageRegions.tsx

```diff
@@ -0,0 +1,49 @@
+import { useTranslatedMicroRegions } from '@ovh-ux/manager-react-components';
+import React, { useEffect, useState } from 'react';
+import { OdsSelect, OdsSpinner } from '@ovhcloud/ods-components/react';
+import { useS3StorageRegions } from '../../api/hook/useRegions';
+import { TRegion } from '../../api/data';
+
+interface S3StorageRegionsProps {
+  projectId: string;
+  onS3StorageRegionChange: (value: string) => void;
+}
+
+export default function S3StorageRegions({
+  projectId,
+  onS3StorageRegionChange,
+}: Readonly<S3StorageRegionsProps>) {
+  const [currentRegion, setCurrentRegion] = useState('');
+
+  const { translateMicroRegion } = useTranslatedMicroRegions();
+  const { data: s3StorageRegions, isLoading } = useS3StorageRegions(projectId);
+
+  useEffect(() => {
+    if (s3StorageRegions?.length) {
+      setCurrentRegion(s3StorageRegions[0].name);
+      onS3StorageRegionChange(s3StorageRegions[0].name);
+    }
+  }, [s3StorageRegions]);
+
+  if (isLoading || !currentRegion) {
```

Comment:
Testing on currentRegion will set a spinner if we don't have any region, is it possible ?

Reply:
i don't believe it's a case that should happen since storages have at least one region